src/gluonts/block/scaler.py

```diff
@@ -30,12 +30,16 @@ class Scaler(nn.HybridBlock):
     ----------
     keepdims
         toggle to keep the dimension of the input tensor.
+    input_NTC
```

Comment:
I think it would be best to configure these blocks directly with `axis`? And default that to 1 for backward compat